src/components/proofEditor/issueInformation.tsx

```diff
@@ -7,14 +8,18 @@ export interface Props {
 	issue: Issue;
 }
 
-export class IssueInformation extends React.Component<Props> {
+export class IssueInformation extends React.Component<Props, {}> {
 
 	public render() {
 
 		let displayedPosition: any = "";
 		const position = this.props.issue.position;
 		if (position) {
-			displayedPosition = <IssuePositionInformation position={position!} />;
```

Comment:
Is position now a boolean or an IssuePositionInformation?

Also; What does `position!` do? I know `!position` which negates a boolean expression.
Does it do the same in the postfix notation?

Reply:
The ! after the variable/function etc. means that this variable/return won't be null or undefined. This is for the typescript compiler since it sometimes can be impossible for an object to become undefined, but the compiler won't recognize it and throw an error.